stripe-ui-core/src/test/java/com/stripe/android/uicore/elements/PhoneNumberFormatterTest.kt

```diff
@@ -104,6 +104,22 @@ internal class PhoneNumberFormatterTest {
         assertThat(formatter.toE164Format("0137111222")).isEqualTo("+44137111222")
     }
 
+    @Test
+    fun `When pattern is missing, there is no formatting`() {
+        val formatter = PhoneNumberFormatter.WithRegion(
+            PhoneNumberFormatter.Metadata(
+                "prefix",
+                "regionCode",
+                // no pattern specified
```

Comment:
Might be worth adding argument names here.

```
PhoneNumberFormatter.Metadata(prefix = "prefix", regionCode = "regionCode", pattern = null)
```

Reply:
good idea, done